pkg/asset/templates/kubeconfig.yaml

```diff
@@ -6,10 +6,11 @@ clusters:
     server: {{ .Server }}
     certificate-authority-data: {{ .CACert }}
```

Comment:
Control plane nodes need to be provisioned with the generated kubeconfig file beforehand. In `hack` that's done by appending the entire file to the cloud-config. On physical machines, it was done by distributing the bearer token (secret material) and Certificate Authority certificate as metadata to template the kubeconfig file. With this change, we'll distribute the certificate authority certificate, client certificate, and client-key (secret material) via metadata (alternately, chuck the entire file in metadata, but that can be ugly).

There is no problem to address here, just noting the change and which bits are secret.